pretixprint/app/src/full/java/eu/pretix/pretixprint/ui/EvolisDirectSettingsFragment.kt

```diff
@@ -16,6 +16,7 @@ import com.google.android.material.textfield.TextInputLayout
 import com.zebra.sdk.common.card.enumerations.CardDestination
 import com.zebra.sdk.common.card.enumerations.CardSource
 import eu.pretix.pretixprint.R
+import eu.pretix.pretixprint.Rotation
 import eu.pretix.pretixprint.byteprotocols.LinkOSCard
```

Comment:
This, together with line 30 `val proto = LinkOSCard()`...

Probably not related at all to this PR, but still wrong...